packages/web-app/src/components/common/Contribution/Contribution.jsx

```diff
@@ -25,6 +28,9 @@ const Contribution = ({
         <AuthorAndDate author={reviewer} date={dateReviewed} verb="Updated" />
       </div>
     )}
+    <a href={`/ui/${type}/${id}/snapshots`}>
```

Comment:
Use the `<Link>` component instead: you should almost never write basic html tags such as p, a or h2 for example.